README.md

```diff
@@ -217,6 +218,20 @@ Generally, if you want to release a prerelease of a repository, and it is an NPM
 
 Please note that in case of a prerelease the `sync-semver-tags` input will be treated as `false`, even if it's set to `true`. This because we don't want to update the main version tags to the latest prerelease commit but only to the latest official release.
 
+## Provenance
+
+NPM will [generate a provenance statement](https://docs.npmjs.com/generating-provenance-statements) for your releases if the following conditions are met:
+
+ - In the `.yml` file that configures your release action:
+   - Add `provenance: true` to your list of **inputs**
+   - Add `id-token: write` to your list of **permissions**
+ - Ensure your CI runner uses NPM >= 9.5.0 (should be the default if Node >= 18)
```

Comment:
The current approach appears to have been that we _do_ delegate this responsibility to the user, but we provide them with a working example config to follow (a `.github/workflows/release.yml` containing `runs-on: ubuntu-latest`). The readme said:

> It is important to be aware that you are responsible for:
> 
> 1. The context where the command is executed. <br /> The command will be executed using the current [GitHub runner](https://github.com/actions/runner#github-actions-runner) configuration (e.g. the one set on `runs-on`).
> You can customize it by...

But we can probably drop or amend this line since our recommended image [already provides NPM 9.5.1](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md), so they'll already get a valid version unless they deviated from our guidance. If they did for some reason, we give an informative error message, then beyond that, it's on them.